doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -542,6 +548,14 @@ Enabling and disabling notations
 
       Scopes are not compatible with :ref:`abbreviations <Abbreviations>`.
 
+   .. example:: Enabling and disabling notations
+
+   .. coqtop:: all
+
+      Disable Notation "+" (all).
+      Enable Notation "_ + _" (all) : type_scope.
+      Disable Notation "x + y" := (sum x y).
```

Comment:
This should be indented (this is what is triggering the CI issue):
```suggestion
      .. coqtop:: all

         Disable Notation "+" (all).
         Enable Notation "_ + _" (all) : type_scope.
         Disable Notation "x + y" := (sum x y).
```

Reply:
Yes, I'm fixing it